tests/pools/abstract/abstract-pool.test.js

```diff
@@ -137,7 +137,7 @@ describe('Abstract pool test suite', () => {
     for (let i = 0; i < numberOfWorkers * 2; i++) {
       promises.push(pool.execute({ test: 'test' }))
     }
-    expect(poolBusy).toBe(numberOfWorkers)
+    expect(poolBusy).toBe(numberOfWorkers + 1)
```

Comment:
**question:** @jerome-benoit Could you write (at least) a short explanation **why** we need to add here 1 on top of `numberOfWorkers`?
I currently don't understand it just by reading the code within the review :slightly_smiling_face:

Reply:
Done.